content/network/drivers/bridge.md

```diff
@@ -105,14 +105,15 @@ flag.
 The following table describes the driver-specific options that you can pass to
 `--option` when creating a custom network using the `bridge` driver.
 
-| Option                                           | Default        | Description                                                 |
-| ------------------------------------------------ | -------------- | ----------------------------------------------------------- |
-| `com.docker.network.bridge.name`                 |                | Interface name to use when creating the Linux bridge.       |
-| `com.docker.network.bridge.enable_ip_masquerade` | `true`         | Enable IP masquerading.                                     |
-| `com.docker.network.bridge.enable_icc`           | `true`         | Enable or Disable inter-container connectivity.             |
-| `com.docker.network.bridge.host_binding_ipv4`    |                | Default IP when binding container ports.                    |
-| `com.docker.network.driver.mtu`                  | `0` (no limit) | Set the containers network Maximum Transmission Unit (MTU). |
-| `com.docker.network.container_iface_prefix`      | `eth`          | Set a custom prefix for container interfaces.               |
+| Option                                           | Default        | Description                                                                                  |
+| ------------------------------------------------ | -------------- | -------------------------------------------------------------------------------------------- |
+| `com.docker.network.bridge.name`                 |                | Interface name to use when creating the Linux bridge.                                        |
+| `com.docker.network.bridge.enable_ip_masquerade` | `true`         | Enable IP masquerading.                                                                      |
+| `com.docker.network.bridge.enable_icc`           | `true`         | Enable or Disable inter-container connectivity.                                              |
+| `com.docker.network.bridge.host_binding_ipv4`    |                | Default IP when binding container ports.                                                     |
+| `com.docker.network.driver.mtu`                  | `0` (no limit) | Set the containers network Maximum Transmission Unit (MTU).                                  |
+| `com.docker.network.container_iface_prefix`      | `eth`          | Set a custom prefix for container interfaces.                                                |
+| `com.docker.network.bridge.inhibit_ipv4`         | `false`        | Prevent Docker from [assigning a gateway IP](#skip-ip-gateway-configuration) to the network. |
```

Comment:
Ah, previous version was mentioning 'unmanaged bridge'. I think it'd be great to better document that feature and make the section about 'inhibit_ipv4' part of it.

Reply:
yeah, I guess that's what I had in mind when I started writing, but then I realized all I'm documenting now is the IP address configuration, so I had to constrain the scope of my section slightly. Maybe later!